pkg/sources/github/github.go

```diff
@@ -788,16 +788,15 @@ func (s *Source) addOrgsByUser(ctx context.Context, user string) {
 }
 
 func (s *Source) addMembersByOrg(ctx context.Context, org string) error {
-	opts := &github.ListOptions{
-		PerPage: membersAppPagination,
-	}
-	optsOrg := &github.ListMembersOptions{
-		PublicOnly:  false,
-		ListOptions: *opts,
```

Comment:
Good find!

The dereference here means `ListOptions` is set to the value of `opts` at this point; if `opts` changes in the future `ListOptions` does not.